package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "skinsback-sdk",
```

Comment:
why such changes?

Reply:
[solved the problem](https://github.com/shaitan-masters/skinsback-sdk/pull/3/commits/913dabcb9c165085306e961a30e6689f776816a6)